### ISSUE TITLE
Organized by Release type

### ISSUE THREAD

**Issue by rohanpandula** [XanderStrike/GazelleUI#2]
It would be awesome if it was organized by 
- Single
- Album
- EP
- Remixes
  etc

its a little hard to sort through them this way

**Comment by XanderStrike** [comment]
Have you tried using the filters under the artist information?

![screen shot 2016-01-14 at 11 56 10 am](https://cloud.githubusercontent.com/assets/1565303/12335804/dc08fdb6-bab5-11e5-8b2a-78d52c90bf33.png)

You can pick an element in this dropdown and it'll filter the releases by that type (and you can further filter them by title with the text field). Is this what you're looking for? Is there anything I could do to make this easier to find?

**Comment by rohanpandula** [comment]
Oh man this is exactly what I was looking for! Thank You.

To make it easier for other people: 
I think the way What.cd default theme (I'm not sure about the other ones) lays everything out with formatting is best:

Release Type
   -> Album
       -> Versions of Albums (organized by date)
   -> EPs
       -> Versions of EPs (organized by date)
   -> Singles
       -> Versions of Singles (organized by date)


Where the release order/date only matters under each category. 


As you can see for Diplo, the first thing that comes up is a mix one of his song was featured rather than his albums or Singles (which is usually the most wanted when looking at a artists (especially for DJs) page).

**Comment by XanderStrike** [comment]
I agree, emulating What's layout is probably the best bet. Right now we're pretty much just sprucing up the output from the API and throwing it on the page, so a little more organization would be nice.
